node/src/test/scala/com/wavesplatform/network/BlacklistParallelSpecification.scala

```diff
@@ -34,24 +34,24 @@ class BlacklistParallelSpecification extends FeatureSpec with GivenWhenThen with
     val address3 = new InetSocketAddress(host3, 2)
 
     def isBlacklisted(address: InetSocketAddress): Boolean =
-      peerDatabase.blacklistedHosts.contains(address.getAddress)
+      peerDatabase.isBlacklisted(address.getAddress)
 
     Scenario("Peer blacklist another peer") {
 
       Given("Peer database is empty")
       assert(peerDatabase.knownPeers.isEmpty)
-      assert(peerDatabase.blacklistedHosts.isEmpty)
+      assert(peerDatabase.detailedBlacklist.isEmpty)
 
       When("Peer adds another peer to knownPeers")
       peerDatabase.touch(address1)
       assert(peerDatabase.knownPeers.contains(address1))
-      assert(!peerDatabase.blacklistedHosts.contains(host1))
+      assert(!peerDatabase.detailedBlacklist.contains(host1))
```

Comment:
```suggestion
      assert(!peerDatabase.isBlacklisted(host1))
```
As it was changed in other places 🤷